partiql-ast/src/main/java/org/partiql/ast/AstVisitor.java

```diff
@@ -61,6 +68,50 @@ public R visitStatement(Statement node, C ctx) {
         return node.accept(this, ctx);
     }
 
+    //
+    // DDL
+    //
+    public R visitDdl(Ddl node, C ctx) {
+        return defaultVisit(node, ctx);
+    }
+
+    public R visitCreateTable(CreateTable node, C ctx) {
+        return defaultVisit(node, ctx);
+    }
+
+    public R visitColumnDefinition(ColumnDefinition node, C ctx) {
+        return defaultVisit(node, ctx);
+    }
+
+    public R visitUnique(TableConstraint.Unique node, C ctx) {
+        return defaultVisit(node, ctx);
+    }
+
+    public R visitNullable(AttributeConstraint.Null node, C ctx) {
+        return defaultVisit(node, ctx);
+    }
+
+    public R visitKeyValue(KeyValue node, C ctx) {
+        return defaultVisit(node, ctx);
+    }
+
+    public R visitPartitionBy(PartitionBy node, C ctx) {
+        return defaultVisit(node, ctx);
+    }
+
+    public R visitUnique(AttributeConstraint.Unique node, C ctx) {
+        return defaultVisit(node, ctx);
+    }
+
+    public R visitCheck(AttributeConstraint.Check node, C ctx) {
+        return defaultVisit(node, ctx);
+    }
```

Comment:
nit: could consider using the full name in the `visit*`. E.g. `visitUnique` -> `visitAttributeConstraintUnique` or if you can think of an appropriate way to shorten.

It might be confusing if some of the function names clash.